src/main/java/seedu/address/ui/jobs/AddDeliveryJobWindow.java

```diff
@@ -61,8 +67,70 @@ public class AddDeliveryJobWindow extends UiPart<Stage> {
     @FXML
     private Button createButton;
     @FXML
+    private Button editButton;
+    @FXML
     private VBox outputErrorPlaceholder;
 
+    // adapted from:
+    // https://stackoverflow.com/questions/26831978/javafx-datepicker-getvalue-in-a-specific-format
+    private StringConverter<LocalDate> deliveryDateConverter = new StringConverter<LocalDate>() {
+        private String pattern = "yyyy-MM-dd";
+        private DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern(pattern);
+
+        {
+            inputDeliveryDate.setPromptText(pattern.toLowerCase());
+        }
+
+        @Override
+        public String toString(LocalDate date) {
+            if (date != null) {
+                return dateFormatter.format(date);
+            } else {
+                return "";
+            }
+        }
+
+        @Override
+        public LocalDate fromString(String string) {
```

Comment:
i think maybe can change this function name to something that's a little easier to understand ex: formString() or getString() instead